src/inmanta/env.py

```diff
@@ -508,6 +508,7 @@ def create_log_content_files(title: str, files: list[str]) -> list[str]:
             not_found: list[str] = []
             conflicts: list[str] = []
             indexes: str = ""
+            no_index: bool = "--no-index" in cmd
```

Comment:
It's also possible to set `--no-index` through an environment variable.

Reply:
Do you know where all the pip env vars are documented? I see in is mentioned at some places and when I try it it does work but I don't find any doc about it.

Reply:
I think every cli option can be set using an environment variable by prefixing the option with `PIP_`, capitalizing all the characters and replacing all the `-` chars with `_`.

Reply:
https://pip.pypa.io/en/stable/topics/configuration/#environment-variables